charts/radar-cloudnative-postgresql/values.yaml

```diff
@@ -1,6 +1,18 @@
-# -- Random database password secrets will be created for these users.
-# @default -- check values.yaml
+# -- Values for creating the database user secretsl
 secrets:
```

Comment:
I would consider renaming this one to `secret` for consistency with this section naming in other charts, e.g. in radar-self-enrollment, as it could lead to some copy-paste mistakes. Just a suggestion though